invenio_rdm_records/contrib/notes/__init_.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+"""Notes contrib module."""
+from .custom_fields import (
```

Comment:
I am not sure why this import is failing in tests.

Reply:
This is because of the file name, it's missing a final `_` 👀

Reply:
Thanks! Classic rookie mistake 😄